internal/api/attestationconfigapi/cli/main.go

```diff
@@ -138,20 +130,34 @@ func runCmd(cmd *cobra.Command, _ []string) (retErr error) {
 		return fmt.Errorf("creating client: %w", err)
 	}
 
-	if err := client.UploadAzureSEVSNPVersion(ctx, inputVersion, flags.uploadDate); err != nil {
-		return fmt.Errorf("uploading version: %w", err)
+	url := "https://d33dzgxuwsgbpw.cloudfront.net"
+	latestAPIVersionAPI, err := attestationconfigapi.NewFetcherWithCustomCDNAndCosignKey(url, constants.CosignPublicKeyDev).FetchAzureSEVSNPVersionLatest(ctx)
+	if err != nil {
+		if errors.Is(err, attestationconfigapi.ErrNoVersionsFound) {
+			log.Infof("No versions found in API, but assuming that we are uploading the first version.")
+		} else {
+			return fmt.Errorf("fetching latest version: %w", err)
+		}
+	}
```

Comment:
```suggestion
	switch {
		case errors.Is(err, attestationconfigapi.ErrNoVersionsFound):
			log.Infof("No versions found in API, but assuming that we are uploading the first version.")
		case err != nil: 
			return fmt.Errorf("fetching latest version: %w", err)
	}
```

just for inspiration. no need to put this in.